lib/backend/syncersv1/updateprocessors/networkpolicyprocessor.go

```diff
@@ -58,11 +59,22 @@ func convertPolicyV2ToV1Spec(spec apiv2.PolicySpec) (interface{}, error) {
 		erules = append(erules, RuleAPIV2ToBackend(erule))
 	}
 
+	// If this policy is namespaced, then add a namespace selector.
+	selector := spec.Selector
+	if ns != "" {
+		nsSelector := fmt.Sprintf("%s == '$s'", apiv2.NamespaceLabel, ns)
```

Comment:
making a note: we're going to need corresponding changes to the CNI plugin to ensure it's writing the correct labels.

Reply:
This should be a `%s` not a `$s`